client/elements/addons/sc-static-page.html

```diff
@@ -15,28 +15,37 @@
                   currentId: {
                       type: String,
                       value: ''
+                  },
+                  language: {
+                      type: String,
+                      statePath: 'siteLanguage',
+                      observer: '_siteLanguageChanged'
+                  },
+                  fallbackLanguage: {
+                      type: String,
+                      value: 'en'
+                  },
+                  languageStringPath: {
+                      type: String
                   }
               }
           }
 
           connectedCallback() {
-            super.connectedCallback();
-          }
-
-          ready() {
-              super.ready();
+              super.connectedCallback();
+              this.loadResources(this.resolveUrl(`${this.languageStringPath}${this.fallbackLanguage}.json`), this.fallbackLanguage, true);
               try {
                   this.shadowRoot.querySelector('main').addEventListener('click', () => {
                       setTimeout(() => {
                           this.currentId = this._scrollToSection(window.location.hash, this.currentId);
                       }, 0);
                   });
               } catch (e) {
-                  console.error(e);
```

Comment:
If we don't print the error message, this becomes a silently passing error, which makes debugging really difficult in the future.

Reply:
Maybe, but it fires at every page startup. I think maybe I should make it an if-statement instead.

Reply:
What's the error message?